expo-module.config.json

```diff
@@ -1,9 +1,9 @@
 {
   "platforms": [
-    "android",
```

Comment:
👍 I suspected this could be the culprit, thanks!

Reply:
sorry for introducing the issue before and thanks for publishing the fix 🚀